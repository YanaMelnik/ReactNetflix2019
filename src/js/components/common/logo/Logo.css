.logo {
  font-weight: 700;
  color: #f34c70;
  margin-left: 30px;
  line-height: 50px;
}
