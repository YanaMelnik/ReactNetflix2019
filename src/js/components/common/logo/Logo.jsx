import React from 'react';

import './Logo.css';

const Logo = () => <div
    className='logo wrapper'>
    netflixroulette
  </div>;

export default Logo;
