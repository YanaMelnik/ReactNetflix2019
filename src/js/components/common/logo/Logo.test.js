import React from 'react';
import { shallow } from 'enzyme';
import Logo from './Logo';

describe('Logo', () => {
    it('should render component correctly in case when all props defined', () => {
        const component = shallow(<Logo/>);

        expect(component).toMatchSnapshot();
    });

    it('should show site name correctly', () => {
        const name = 'netflixroulette';
        const component = shallow(<Logo/>);

        expect(component.matchesElement(<div>{name}</div>)).toBeTruthy();
    });

    it('should show logo with correct classes', () => {
        const component = shallow(<Logo/>);

        expect(component.find('div.logo.wrapper')).toHaveLength(1);
    });
});
