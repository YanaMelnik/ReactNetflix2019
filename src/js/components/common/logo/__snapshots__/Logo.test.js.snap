// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render component correctly in case when all props defined 1`] = `
<div
  className="logo wrapper"
>
  netflixroulette
</div>
`;
